src/index.html

```diff
@@ -11,6 +11,34 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="http://localhost:8080/"
+      method="get"
+    >
+
+      <label for="searchForm1">
+        <input
+          placeholder= "Try &quot;Los Angeles&quot;"
+          type="text"
+          id="myInput"
+          data-qa="big"
```

Comment:
```suggestion
```
Read [Requirements](https://github.com/mate-academy/layout_search-bar-airbnb#requirements)

Reply:
without  data-qa="big" test doesn't pass

Reply:
> without data-qa="big" test doesn't pass

Read Requirements - you need to put it in right place, not remove

Reply:
But if l put it in this way, test doesn't pass:
if:
 <form
      data-qa="big"
    >
<input
     data-qa="keypress"
 >

    <form
      data-qa="small"
    >

Reply:
<form
      data-qa="big"
>
   <input
          data-qa="keypress"
     >
  <form
      data-qa="small"
    >

Reply:
` <form
      data-qa="big"
  >`
<input
          data-qa="keypress"
 >
 <form
      data-qa="small"
    >

Reply:
Sorry, but if l put it in this way, test doesn't pass, l don't know why:
`<form
      action="/"
      method="get"
      data-qa="big"
    >
 <input
          data-qa="keypress"
        >
 <form
      action="/"
      method="get"
      data-qa="small"
    >`